test/fileSummaryV1/integration_fileSummaryV1.test.js

```diff
@@ -0,0 +1,160 @@
+/** Run integration tests from the command line.  For example:
+ *
+ *    BASE_URL="https://us-central1-broad-dsde-dev.cloudfunctions.net" npm run-script integration_fileSummaryV1
+ *
+ * Run integration tests after a deployment to confirm that the functions deployed successfully
+ */
+
+const test = require('ava');
+const supertest = require('supertest')(process.env.BASE_URL);
+const assert = require('assert');
+const { GoogleToken } = require('gtoken');
+const { postJsonTo } = require('../../common/api_adapter');
+let unauthorizedToken;
+let authorizedToken;
+
+let fenceAuthLink = 'https://bond-fiab.dsde-dev.broadinstitute.org:31443/api/link/v1/fence/oauthcode?oauthcode=IgnoredByMockProvider&redirect_uri=http%3A%2F%2Flocal.broadinstitute.org%2F%23fence-callback'
+
+test.before(async () => {
+    unauthorizedToken = await new GoogleToken({
+        keyFile: '/Users/mtalbott/ws/broad/martha/firecloud-account.json', // TODO: change filepath to actual .pem key (need it to be .json?)
```

Comment:
I am not sure where to get this key file from, but it should be accessible from within the FiaB so I just need to determine where it is located and adjust this file path accordingly.